client/src/components/dashboard/RecentForumPosts.jsx

```diff
@@ -1,5 +1,6 @@
 import "./RecentForumPosts.css";
```

Comment:
This also doesn't appear to fetch anything from the DB

Reply:
fixed issue